.github/workflows/c-cpp.yml

```diff
@@ -48,6 +48,11 @@ jobs:
       - name: Run LDBC
         if: runner.os == 'Linux'
         run: |
+          cd build
+          rm -rf ./*
+          cmake ..
+          make
+          cd ..
```

Comment:
Why the LDBC benchmark requires rebuilding without `ASAN`?

Reply:
I guess it is because we may want to test build both release and debug?

Reply:
It must be linked with `-fsanitize=address`, if `libgrape-lite.so` is compiled with ASAN, and vice versa.
Controlling CMake options of LDBC executable is not out of the box, and it's algorithm is already covered in `app_test.sh`, so getting a clean build has the least impact on the codebase, still remains desired functionality.

Reply:
@siyuan0322 
Sorry, I am not sure if I understand your reply correctly. The rebuild is to avoid its interference with LDBC driver?
If it is the case, please revise your PR:
1. Remove the CMakeLists.txt under ldbc_driver, and add the config to the the main one.
2. Make sure re-build to be DEBUG, or it might break the coverage?